apisix/cli/ngx_tpl.lua

```diff
@@ -833,6 +840,32 @@ http {
             proxy_pass $upstream_mirror_uri;
         }
         {% end %}
+
+        {% if enabled_plugins["proxy-mirror"] then %}
+        location = /proxy_mirror_grpc {
+            internal;
+
+            {% if not use_apisix_base then %}
+            if ($upstream_mirror_uri = "") {
+                return 200;
+            }
+            {% end %}
+
+
+            {% if proxy_mirror_timeouts then %}
+                {% if proxy_mirror_timeouts.connect then %}
+            proxy_connect_timeout {* proxy_mirror_timeouts.connect *};
```

Comment:
Should use `grpc_*_timeout`
http://nginx.org/en/docs/http/ngx_http_grpc_module.html#grpc_connect_timeout

Reply:
updated.